README.md

```diff
@@ -1,2 +1,31 @@
+|-----------------------------|--------------------------------------------------------------------------------------------------------|------------------|--------------------------------------------|
+| IEXEC_RESULT_PROXY_PORT     | Server port of the result proxy.                                                                       | Positive integer | 13200                                      |
+| MONGO_HOST                  | Host to connect to the MongoDB. It is recommended to use the same as the one hosting the result proxy. | String           | localhost                                  |
+| MONGO_PORT                  | Port to connect to the MongoDB.                                                                        | Positive integer | 13202                                      |
+| IEXEC_CHAIN_ID              | ID of the chain.                                                                                       | Integer          | 17                                         |
+| IEXEC_IS_SIDECHAIN          | Whether the chain node is a sidechain or an ERC20 token.                                               | Boolean          | false                                      |
+| IEXEC_PRIVATE_CHAIN_ADDRESS | Private URL to connect to the blockchain node.                                                         | String           | http://localhost:8545                      |
+| IEXEC_PUBLIC_CHAIN_ADDRESS  | [unused] Public URL to connect to the blockchain node.                                                 | String           | http://localhost:8545                      |
+| IEXEC_HUB_ADDRESS           | Ethereum address to interact with the iExec smart contracts.                                           | String           | 0xBF6B2B07e47326B7c8bfCb4A5460bef9f0Fd2002 |
+| IEXEC_START_BLOCK_NUMBER    | [Unused]                                                                                               | Positive integer | 0                                          |
+| IEXEC_GAS_PRICE_MULTIPLIER  | Transactions will be sent with `networkGasPrice * IEXEC_GAS_PRICE_MULTIPLIER`.                         | Float            | 1.0                                        |
+| IEXEC_GAS_PRICE_CAP         | In Wei, will be used for transactions if `networkGasPrice * IEXEC_GAS_PRICE_MULTIPLIER > gasPriceCap`. | Integer          | 22000000000                                |
+| IEXEC_IPFS_HOST             | Host to connect to the IPFS node.                                                                      | String           | 127.0.0.1                                  |
+| IEXEC_IPFS_PORT             | Server port of the IPFS node.                                                                          | Positive integer | 5001                                       |
```

Comment:
```suggestion
|Environment variables|Description|Values|Default value|
|-|-|-|-|
|IEXEC_RESULT_PROXY_PORT|Server port of the result proxy.|Positive integer|13200|
|MONGO_HOST|Host to connect to the MongoDB. It is recommended to use the same as the one hosting the result proxy.|String|localhost|
|MONGO_PORT|Port to connect to the MongoDB.|Positive integer|13202|
|IEXEC_CHAIN_ID|ID of the chain.|Integer|17|
|IEXEC_IS_SIDECHAIN|Whether the chain node is a sidechain or an ERC20 token.|Boolean|false|
|IEXEC_PRIVATE_CHAIN_ADDRESS|Private URL to connect to the blockchain node.|String|http://localhost:8545|
|IEXEC_PUBLIC_CHAIN_ADDRESS|[unused] Public URL to connect to the blockchain node.|String|http://localhost:8545|
|IEXEC_HUB_ADDRESS|Ethereum address to interact with the iExec smart contracts.|String|0xBF6B2B07e47326B7c8bfCb4A5460bef9f0Fd2002|
|IEXEC_START_BLOCK_NUMBER|[Unused]|Positive integer|0|
|IEXEC_GAS_PRICE_MULTIPLIER|Transactions will be sent with `networkGasPrice * IEXEC_GAS_PRICE_MULTIPLIER`.|Float|1.0|
|IEXEC_GAS_PRICE_CAP|In Wei, will be used for transactions if `networkGasPrice * IEXEC_GAS_PRICE_MULTIPLIER > gasPriceCap`.|Integer|22000000000|
|IEXEC_IPFS_HOST|Host to connect to the IPFS node.|String|127.0.0.1|
|IEXEC_IPFS_PORT|Server port of the IPFS node.|Positive integer|5001|
```

Reply:
It is intellij auto-formatting of markdown tables.
It triggers itself automatically.
And the preview mode within intellij is reaaly nice to have.

Reply:
Need to disable Markdown auto-assistance in settings to solve the formatting point